[dogstatsd] enable the no-aggregation pipeline. (#13440)
DataDog/datadog-agent: docs/dogstatsd/internals.md

    * If `dogstatsd_pipeline_count` has a value, the number of TimeSampler pipelines equals that value.
    * If neither condition above is true, one TimeSampler pipeline runs.

## NoAggregationStreamWorker

Input: slice of MetricSamples

> The NoAggregationStreamWorker runs an infinite loop in a goroutine, it receives metric sample with timestamp and batches them to send them as fast as possible to the intake. It performs no aggregation nor extra processing except from adding tags to the metrics.

Reviewer: ```suggestion
The NoAggregationStreamWorker runs an infinite loop in a goroutine. It receives metric samples with timestamps, and it batches them to be sent as quickly as possible to the intake. It performs no aggregation nor extra processing, except from adding tags to the metrics.
```

It is running only when `dogstatsd_no_aggregation_pipeline` is set to `true`.

> Payload it is sending to the intake (through the normal `Serializer`/`Forwarder` pieces) contains at maximum `dogstatsd_no_aggregation_pipeline_batch_size` metrics, default is `256`.

Reviewer: ```suggestion
The payload being sent to the intake (through the normal `Serializer`/`Forwarder` pieces) contains, at maximum, `dogstatsd_no_aggregation_pipeline_batch_size` metrics. This value defaults to `256`.
```
